src/Stratis.Bitcoin.Features.BlockStore/BlockStoreFeature.cs

```diff
@@ -123,7 +123,7 @@ public override Task InitializeAsync()
             }
 
             // Signal to peers that this node can serve blocks.
-            this.connectionManager.Parameters.Services = (this.storeSettings.Prune != 0 ? NetworkPeerServices.Nothing : NetworkPeerServices.Network);
+            this.connectionManager.Parameters.Services = (this.storeSettings.PruningEnabled ? NetworkPeerServices.Nothing : NetworkPeerServices.Network);
```

Comment:
Please add a TODO here to add NetworkLimited which is what BTC uses for pruned nodes.

Reply:
Fixed